src/views/accompagnement/liste.vue

```diff
@@ -227,14 +227,21 @@ export default {
         ? `/api/followups/id/${this.$route.params.surveyId}`
         : `/api/followups/surveys`
       try {
-        const response = await fetch(uri, {
-          method: "GET",
-          redirect: "error",
-          headers: {
-            Accept: "application/json",
-            "Content-Type": "application/json",
-          },
-        })
+        const response = await fetch(
+          `${uri}?${Math.random().toString(36).substr(2, 5)}`,
```

Comment:
La logique est dupliqué : 
```suggestion
          uri,
```
```
params: {
              // Ajoute un date pour éviter la mise en cache du serveur
              t: new Date().getTime(),
            },
            ```

Reply:
Bien vu